Services/Form/templates/default/tpl.prop_datetime.html

```diff
@@ -1,7 +1,8 @@
 <div class="ilFormInnerCol">
 <div class="form-group form-inline">
 	<div class="input-group date" id="{DATEPICKER_ID}">
-		<input type="text" class="form-control" placeholder="{PLACEHOLDER}" style="min-width:120px" value="{DATEPICKER_VALUE}" name="{DATE_ID}"{DATEPICKER_DISABLED} {REQUIRED} />
+		<input aria-describedby="date_desc" type="text" class="form-control" placeholder="{PLACEHOLDER}" style="min-width:120px" value="{DATEPICKER_VALUE}" name="{DATE_ID}"{DATEPICKER_DISABLED} {REQUIRED} />
+		<p id="date_desc" style="display:none">{DESCRIPTION}</p>
```

Comment:
Why using disaply:none here? Afaik this prevents some screen readers from reading the text at all. Bootstrap has a class sr-only for these cases.